### ISSUE TITLE
Add to new user form in some way

### ISSUE THREAD

**Issue by MikeCook9994** [UW-UPL/TourGuide#1]
While I wouldn't suggest mandating it be done, the user should indicate they've been made aware of its existence so it doesn't fall to the side one day.

Opinions on the best way to approach this? Perhaps a quick blurb about it and a check box to indicate they read some blurb. Or would that be as pointless as I feel like it would be?

**Comment by LOZORD** [comment]
Yeah, that sounds good. I think we could also automatically add it (or an executable version) when their new user (`~`) directory is created (and then maybe link to the Desktop so they see it).

We could just pull the raw files from GitHub in the config script.

**Comment by spwilson2** [comment]
Could add a box to the new user form saying that they're aware of the
script as well.

**Comment by MikeCook9994** [comment]
That's what I thought was the best idea @spwilson2 That way we don't have to remember to tell them about it upon registering. I feel like most people will read the new user form.
